lib/internal/util/comparisons.js

```diff
@@ -322,7 +335,7 @@ function objectComparisonStart(val1, val2, mode, memos) {
         return false;
       }
     } else if (mode === kLoose &&
-               (isFloat32Array(val1) || isFloat64Array(val1) || isFloat16Array(val1))) {
+              (isFloat32Array(val1) || isFloat64Array(val1) || isFloat16Array(val1))) {
```

Comment:
unrelated change (we should probably have the linter enforce one or the other)
```suggestion
               (isFloat32Array(val1) || isFloat64Array(val1) || isFloat16Array(val1))) {
```